uni/lib/view/restaurant/widgets/restaurant_slot.dart

```diff
@@ -38,14 +38,16 @@ class RestaurantSlot extends StatelessWidget {
     );
   }
 
-  Widget createCantineSlotType(context) {
+  Widget createCanteenSlotType(context) {
     final mealsType = type.toLowerCase();
 
     String icon;
-    if (mealsType.contains("carne")) {icon = 'assets/icons-cantines/chicken.svg';}
-    else if (mealsType.contains("peixe")) {icon = 'assets/icons-cantines/fish.svg';}
-    else if (mealsType.contains("vegetariano")) {icon = 'assets/icons-cantines/salad.svg';}
-    else if (mealsType.contains("dieta")) {icon = 'assets/icons-cantines/diet.svg';}
+    if (mealsType.contains("sopa")) {icon = 'assets/meal-icons/soup.svg';}
+    else if (mealsType.contains("carne")) {icon = 'assets/meal-icons/chicken.svg';}
+    else if (mealsType.contains("peixe")) {icon = 'assets/meal-icons/fish.svg';}
+    else if (mealsType.contains("dieta")) {icon = 'assets/meal-icons/diet.svg';}
+    else if (mealsType.contains("vegetariano")) {icon = 'assets/meal-icons/vegetarian.svg';}
+    else if (mealsType.contains("salada")) {icon = 'assets/meal-icons/salad.svg';}
```

Comment:
should this be a switch?

Reply:
It can't, but using:
```
  static const mealTypeIcons = {
    'sopa': 'assets/meal-icons/soup.svg',
    'carne': 'assets/meal-icons/chicken.svg',
    'peixe': 'assets/meal-icons/fish.svg',
    'dieta': 'assets/meal-icons/diet.svg',
    'vegetariano': 'assets/meal-icons/vegetarian.svg',
    'salada': 'assets/meal-icons/salad.svg',
  };
```
You can simply:
```
  Widget createCanteenSlotType(context) {
    final mealsType = type.toLowerCase();
    final icon = mealTypeIcons.entries
        .firstWhere((element) => mealsType.contains(element.key),
            orElse: () => const MapEntry('', ''))
        .value;
    return Tooltip(
        message: type,
        child: icon != ''
            ? SvgPicture.asset(
                icon,
                color: Theme.of(context).primaryColor,
                height: 20,
              )
            : null);
  }
```